Can't use `-include-runtime` with `elide kotlinc`

### Expected Behavior

1) Compile some Kotlin with Elide
2) Use a CLI tool invocation
3) Include the `-include-runtime` flag
4) It works

### Actual Behaviour

Crash

### Stacktrace

```
$ elide kotlinc -- ... -include-runtime ...
cannot extract '/org/jetbrains/kotlin/utils/PathUtil.class' from 'resource:/org/jetbrains/kotlin/utils/PathUtil.class'
[kotlinc:exception] java.lang.IllegalStateException: Resource not found: /org/jetbrains/kotlin/utils/PathUtil.class
        at org.jetbrains.kotlin.utils.PathUtil.getResourcePathForClass(PathUtil.kt:164)
        at org.jetbrains.kotlin.utils.PathUtil.getPathUtilJar(PathUtil.kt:159)
        at org.jetbrains.kotlin.utils.PathUtil.getKotlinPathsForCompiler(PathUtil.kt:119)
        at org.jetbrains.kotlin.cli.jvm.compiler.CompileEnvironmentUtil.writeRuntimeToJar(CompileEnvironmentUtil.java:138)
        at org.jetbrains.kotlin.cli.jvm.compiler.CompileEnvironmentUtil.doWriteToJar(CompileEnvironmentUtil.java:95)
        at org.jetbrains.kotlin.cli.jvm.compiler.CompileEnvironmentUtil.writeToJar(CompileEnvironmentUtil.java:123)
        at org.jetbrains.kotlin.cli.jvm.compiler.CliCompilerUtilsKt.writeOutput(cliCompilerUtils.kt:83)
        at org.jetbrains.kotlin.cli.jvm.compiler.CliCompilerUtilsKt.writeOutputsIfNeeded(cliCompilerUtils.kt:119)
        at org.jetbrains.kotlin.cli.pipeline.jvm.JvmWriteOutputsPhase.executePhase(JvmWriteOutputsPhase.kt:17)
        at org.jetbrains.kotlin.cli.pipeline.jvm.JvmWriteOutputsPhase.executePhase(JvmWriteOutputsPhase.kt:12)
        at org.jetbrains.kotlin.cli.pipeline.PipelinePhase.phaseBody(PipelinePhase.kt:68)
        at org.jetbrains.kotlin.cli.pipeline.PipelinePhase.phaseBody(PipelinePhase.kt:58)
        at org.jetbrains.kotlin.config.phaser.NamedCompilerPhase.invoke(CompilerPhase.kt:102)
        at org.jetbrains.kotlin.backend.common.phaser.CompositePhase.invoke(PhaseBuilders.kt:22)
        at org.jetbrains.kotlin.config.phaser.CompilerPhaseKt.invokeToplevel(CompilerPhase.kt:53)
        at org.jetbrains.kotlin.cli.pipeline.AbstractCliPipeline.runPhasedPipeline(AbstractCliPipeline.kt:109)
        at org.jetbrains.kotlin.cli.pipeline.AbstractCliPipeline.execute(AbstractCliPipeline.kt:68)
        at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecutePhased(K2JVMCompiler.kt:79)
        at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecutePhased(K2JVMCompiler.kt:45)
        at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:90)
        at org.jetbrains.kotlin.cli.common.CLICompiler.exec(CLICompiler.kt:352)
        at elide.tooling.kotlin.KotlinCompiler.invoke(KotlinCompiler.kt:636)
        at elide.tool.cli.cmd.tool.DelegatedToolCommand.invoke$suspendImpl(DelegatedToolCommand.kt:196)
        at elide.tool.cli.cmd.tool.DelegatedToolCommand.invoke(DelegatedToolCommand.kt)
        at elide.tool.cli.AbstractSubcommand$invoke$2$1.invokeSuspend(AbstractSubcommand.kt:602)
        at elide.tool.cli.AbstractSubcommand$invoke$2$1.invoke(AbstractSubcommand.kt)
        at elide.tool.cli.AbstractSubcommand$invoke$2$1.invoke(AbstractSubcommand.kt)
        at elide.tool.cli.AbstractSubcommand.initializeToolResources(AbstractSubcommand.kt:476)
        at elide.tool.cli.AbstractSubcommand.invoke-AZeniH0$suspendImpl(AbstractSubcommand.kt:600)
        at elide.tool.cli.AbstractSubcommand.invoke-AZeniH0(AbstractSubcommand.kt)
        at elide.tool.cli.AbstractToolCommand$call$$inlined$execute$1.invokeSuspend(AbstractToolCommand.kt:249)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:94)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:70)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:48)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:297)
        at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:57)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at elide.tool.cli.Elide$Companion.entry(Elide.kt:614)
        at elide.tool.cli.MainKt.main(main.kt:389)
        at java.base@24.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)

Failed to call tool: kotlinc
java.lang.IllegalStateException: Failed to call tool: UnspecifiedFailure
	at elide.tool.cli.cmd.tool.DelegatedToolCommand.invoke$suspendImpl(DelegatedToolCommand.kt:172)
	at elide.tool.cli.cmd.tool.DelegatedToolCommand.invoke(DelegatedToolCommand.kt)
	at elide.tool.cli.AbstractSubcommand$invoke$2$1.invokeSuspend(AbstractSubcommand.kt:602)
	at elide.tool.cli.AbstractSubcommand$invoke$2$1.invoke(AbstractSubcommand.kt)
	at elide.tool.cli.AbstractSubcommand$invoke$2$1.invoke(AbstractSubcommand.kt)
	at elide.tool.cli.AbstractSubcommand.initializeToolResources(AbstractSubcommand.kt:476)
	at elide.tool.cli.AbstractSubcommand.invoke-AZeniH0$suspendImpl(AbstractSubcommand.kt:600)
	at elide.tool.cli.AbstractSubcommand.invoke-AZeniH0(AbstractSubcommand.kt)
	at elide.tool.cli.AbstractToolCommand$call$$inlined$execute$1.invokeSuspend(AbstractToolCommand.kt:249)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:94)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:70)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:48)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:297)
	at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:57)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at elide.tool.cli.Elide$Companion.entry(Elide.kt:614)
	at elide.tool.cli.MainKt.main(main.kt:389)
	at java.base@24.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Failed to call tool: UnspecifiedFailure
Failed with exit code 1
make: *** [Makefile:84: /.../.dev/dist] Error 1
```

### Steps To Reproduce

_No response_

### Environment Information

See enclosed

### Runtime

Native

### Language

JVM

### Example Application

_No response_

### Version

1.0.0-beta9

--- a/packages/builder/src/main/kotlin/elide/tooling/jvm/JavaCompiler.kt
+++ b/packages/builder/src/main/kotlin/elide/tooling/jvm/JavaCompiler.kt
@@ -405,6 +405,8 @@ public val javac: Tool.CommandLineTool = Tool.describe(
         !it.startsWith("-") && (it != outSpec)
       }.map {
         Paths.get(it)
+      }.filter {
+        it.exists()
       }
       return Triple(sources.asSequence(), outSpec, argsList)
     }

--- a/packages/cli/src/main/kotlin/elide/tool/cli/cmd/tool/nativeImage/NativeImageAdapter.kt
+++ b/packages/cli/src/main/kotlin/elide/tool/cli/cmd/tool/nativeImage/NativeImageAdapter.kt
@@ -151,7 +151,12 @@ import elide.tooling.project.ElideProject
             listOf(it)
           }
         }
-        val outSpecPositionMinusOne = argsList.indexOf("-H:Path")
+        val outSpecPositionMinusOne = (
+          when (val idx = argsList.indexOf("-H:Path")) {
+            -1 -> argsList.indexOf("-o")
+            else -> idx
+          }
+        )
         val outSpec = if (outSpecPositionMinusOne < 0) {
           // come up with an output path; use `.dev/artifacts/native-image` by default
           // @TODO resolve from config

--- a/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
+++ b/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
@@ -11167,14 +11167,8 @@
     {
       "type": "org.jetbrains.kotlin.cli.common.arguments.CommonToolArguments",
       "allDeclaredFields": true,
-      "methods": [
-        {
-          "name": "setSuppressWarnings",
-          "parameterTypes": [
-            "boolean"
-          ]
-        }
-      ]
+      "allDeclaredMethods": true,
+      "allDeclaredConstructors": true
     },
     {
       "type": "org.jetbrains.kotlin.cli.common.arguments.Freezable",

--- a/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
+++ b/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
@@ -12960,6 +12960,9 @@
   ],
   "resources": [
     {
+      "glob": "/org/jetbrains/kotlin/**/*.class"
+    },
+    {
       "glob": "/META-INF/elide/samples/**/*"
     },
     {

